roxie/ccd/ccdmain.cpp

```diff
@@ -537,6 +525,29 @@ void readStaticTopology()
     createStaticTopology(allRoles, traceLevel);
 }
 
+static constexpr const char * defaultJson = R"!!({
+  "version": "1.0",
+  "Roxie": {
+    "allFilesDynamic": "1",
+    "numChannels": "1",
```

Comment:
minor: Should these be unquoted, and localSlave a boolean?  I know it makes no difference in the current implementation.